### ISSUE TITLE
can be used offline?

### ISSUE THREAD

**Issue by RYG81** [philz1337x/ComfyUI-ClarityAI#1]
can this be used offline?

**Comment by hong8206** [comment]
No, the APi needs to be networked

**Comment by lrq3000** [comment]
Yes the free_workflow.json can be dropped in ComfyUI and it runs offline.

The secret sauce is in the ControlNet:

https://huggingface.co/copybaiter/ControlNet/blob/main/control_v11f1e_sd15_tile.safetensors

The rest can be changed to other models, and the upscaler can be bypassed (it does not work for me, there seems to be a problem in the python code for this node currently).

For more infos on how to manipulate or recreate this workflow, there was a related discussion on reddit: https://www.reddit.com/r/StableDiffusion/comments/136ere0/after_this_tile_controlnet_thingy_i_think_sd_beat/

Note however that this ComfyUI workflow does not have any of the easy sliders such as Creativity. So it's unclear how to recreate exactly how ClarityAI UI works (if somebody knows, please share!).

**Comment by lrq3000** [comment]
I hastily added instructions on how I did it in my fork, and I also updated the workflow to work with the latest version of ComfyUI and with a SD1.5-Hyper model (faster to iterate): https://github.com/lrq3000/ComfyUI-ClarityAI
